src/styles/blocks/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+
+  width: #{$card-width};
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $primary-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 16px;
+    margin-top: 16px;
+  }
+
+  &__reviewes-paragraph {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14;
+    color: $secondary-color;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price-paragraph {
+    font-weight: 400px;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    color: $primary-color;
+  }
+
+  &__buy-button {
+    margin-top: 16px;
+    height: 40px;
+    width: 100%;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16;
+
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```

---

Thx!
Done